src/containers/MyNdla/Arena/components/TopicCard.tsx

```diff
@@ -21,81 +20,91 @@ interface Props {
   count: number;
   locked?: boolean;
 }
-const StyledSafelink = styled(SafeLink)`
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  gap: ${spacing.normal};
-  padding: ${spacing.normal} ${spacing.medium};
-  padding-left: ${spacing.large};
-  border: 1px solid ${colors.brand.light};
-  border-radius: ${misc.borderRadius};
-  box-shadow: none;
-  background-color: ${colors.background.lightBlue};
-  &:hover,
-  &:focus-visible {
-    background-color: ${colors.brand.lighter};
-    [data-name="hover"] {
-      text-decoration: none;
-    }
-  }
-`;
+const StyledSafelink = styled(SafeLink, {
+  base: {
+    alignItems: "center",
+    border: "1px solid",
+    borderColor: "stroke.default",
+    borderRadius: "xsmall",
+    display: "flex",
+    flexDirection: "row",
+    gap: "medium",
+    justifyContent: "space-between",
+    paddingBlock: "medium",
+    paddingInlineStart: "xxlarge",
+    paddingInlineEnd: "large",
 
-const StyledHeader = styled(Text)`
-  color: ${colors.brand.primary};
-  text-decoration: underline;
-  overflow-wrap: break-word;
-  cursor: pointer;
-`;
+    _focusVisible: {
+      backgroundColor: "surface.hover",
+    },
```

Comment:
Tror ikke vi er interessert i å endre bakgrunnsfarge når ting fokuseres lenger.

Reply:
Fant ikke noe oppdatert design som bekrefta hover, så det er bare beholdt fra slik det var 🤷 Har vi noe å bekrefte det mot at slik hover-indikasjon skal bort, også for klikkbare blokkelementer i min ndla?

Reply:
Hover går fint, men vi har fjernet tilnærmet alle fokus-effekter.